_posts/2024-02-26-cache-refresh.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: post
+title:  "A cache refresh for Trino"
+author: Manfred Moser
+image: /assets/blog/trino-cache-refresh.png
+excerpt_separator: <!--more-->
+show_pagenav: true
+---
+
+Leon Bambrick once said "There are only two hard things in computer science:
+cache invalidation, naming things, and off-by-one errors" improving the initial
+wisdom from [Phil Karlton](https://www.karlton.org/2017/12/naming-things-hard/).
+Well, in the Trino community we know all about caching, naming, off-by-one
+errors, and improving on original wisdom. With the recent [Trino 439
+release]({{site.url}}/docs/current/release/release-438.html) caching from object
+storage file systems got a refresh. Users of catalogs using the Delta Lake,
```

Comment:
"Users of catalogs using" sounds kind of redundant, I'd reword.

Reply:
done